opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
```

Comment:
Changing to `peer.*`

I'll hold off on process-guid, as I am not yet sure how that can be captured. In fact, it sounds like something that the tracing system can capture automatically, without participation from instrumentation code. In order to propagate it with in-band context we'd need yet another type of context data, one whose lifespan is one p2p call, as opposed to the lifespan of the whole trace.